std/traits.d

```diff
@@ -4869,7 +4869,9 @@ Determines whether the function type $(D F) is covariant with $(D G), i.e.,
 functions of the type $(D F) can override ones of the type $(D G).
  */
 template isCovariantWith(F, G)
-    if (is(F == function) && is(G == function))
+    if ((is(F == function) && is(G == function)) ||
```

Comment:
Is there a reason you didn't use [isSomeFunction](https://dlang.org/library/std/traits/is_some_function.html) here?

Reply:
I can't say. It 's an old patch found on bugzilla.

Reply:
That being said it seems obvious that the author wanted `F` and `G` to have the same `function membership type` (how to name that ?), so that covariance couldn't be tested with a pointer to static member function and a delegate.